Makefile

```diff
@@ -60,10 +60,11 @@ ARCH	:=	-march=armv6k -mtune=mpcore -mfloat-abi=hard -mtp=soft
 
 # FIXME: Remove -fcommon once all multiple definitions are fixed
 CFLAGS	:=	-g -Wall -O2 -mword-relocations \
+			-Wno-unused-variable \
```

Comment:
There are so many of these warnings right now, and it's hard to know quite what to do with many of them. Silence the warning for now until sometime later when they can be addressed separately.